annorxiver_modules/document_helper.py

```diff
@@ -1,10 +1,15 @@
+import csv
+import lzma
 import re
 
-from gensim.parsing.preprocessing import remove_stopwords
```

Comment:
Very high level question. What was out of date in your current parser?

Reply:
Nothing wrong with using gensim's parser for removing stopwords; however, I switched to using spaCy for processing and they provide their own stopwords. To be consistent I switched.